regression/cbmc/sat-solver/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+test.c
+--sat-solver cadical
+^EXIT=10$
+^SIGNAL=0$
+\[main\.assertion\.1] line \d+ assertion y != x: FAILURE
+\[main\.assertion\.2] line \d+ assertion y <= x: SUCCESS
+^VERIFICATION FAILED$
+--
```

Comment:
Based on the default behaviour for the `lingeling` test above, this will just use another solver if something ir wrong with CaDiCaL, so we're not actually testing that CaDiCaL is used. Can we please match the output for the actual CaDiCaL string, and/or check the failure to detect/use CaDiCaL is not present?

Reply:
I added this line to the `test.desc`:
```
Solving with CaDiCaL|The specified solver, 'cadical', is not available. The default solver will be used instead.
```
Is this sufficient?